prac_07/squaring.kv

```diff
@@ -1,15 +1,20 @@
 BoxLayout:
+    orientation: 'horizontal'
```

Comment:
I think the top level of boxlayout orientation is `'vertical'` by refering the image below in prac_07
![image](https://user-images.githubusercontent.com/79427804/116849440-b679bc00-ac29-11eb-9622-08cfd58adf94.png)